tests/integration/simple_copy_test/test_simple_copy.py

```diff
@@ -1,311 +0,0 @@
-        })
-
-        results = self.run_dbt(["seed"])
-        self.assertEqual(len(results),  1)
-        results = self.run_dbt()
-        self.assertEqual(len(results),  7)
-
-        self.assertManyTablesEqual(["SEED", "VIEW_MODEL", "INCREMENTAL", "MATERIALIZED", "GET_AND_REF"])
-
-        self.use_default_project({
-            "seed-paths": [self.dir("snowflake-seed-update")],
-            "quoting": {"identifier": False},
-        })
-        results = self.run_dbt(["seed"])
-        self.assertEqual(len(results),  1)
-        results = self.run_dbt()
-        self.assertEqual(len(results),  7)
-
-        self.assertManyTablesEqual(["SEED", "VIEW_MODEL", "INCREMENTAL", "MATERIALIZED", "GET_AND_REF"])
-
-        self.use_default_project({
-            "test-paths": [self.dir("tests")],
-            "seed-paths": [self.dir("snowflake-seed-update")],
-            "quoting": {"identifier": False},
-        })
-        self.run_dbt(['test'])
-
-    @use_profile("snowflake")
-    def test__snowflake__seed__quoting_switch(self):
-        self.use_default_project({
-            "quoting": {"identifier": False},
-            "seed-paths": [self.dir("snowflake-seed-initial")],
-        })
-
-        results = self.run_dbt(["seed"])
-        self.assertEqual(len(results),  1)
-
-        self.use_default_project({
-            "seed-paths": [self.dir("snowflake-seed-update")],
-            "quoting": {"identifier": True},
-        })
-        results = self.run_dbt(["seed"], expect_pass=False)
-
-        self.use_default_project({
-            "test-paths": [self.dir("tests")],
-            "seed-paths": [self.dir("snowflake-seed-initial")],
-        })
-        self.run_dbt(['test'])
-
-
-class TestSimpleCopyQuotingIdentifierOn(BaseTestSimpleCopy):
-    @property
-    def project_config(self):
-        return self.seed_quote_cfg_with({
-            'quoting': {
-                'identifier': True,
-            },
-        })
-
-    @use_profile("snowflake")
-    def test__snowflake__simple_copy__quoting_on(self):
-        self.use_default_project({
-            "seed-paths": [self.dir("snowflake-seed-initial")],
-        })
-
-        results = self.run_dbt(["seed"])
-        self.assertEqual(len(results),  1)
-        results = self.run_dbt()
-        self.assertEqual(len(results),  7)
-
-        self.assertManyTablesEqual(["seed", "view_model", "incremental", "materialized", "get_and_ref"])
-
-        self.use_default_project({
-            "seed-paths": [self.dir("snowflake-seed-update")],
-        })
-        results = self.run_dbt(["seed"])
-        self.assertEqual(len(results),  1)
-        results = self.run_dbt()
-        self.assertEqual(len(results),  7)
-
-        self.assertManyTablesEqual(["seed", "view_model", "incremental", "materialized", "get_and_ref"])
-
-        # can't run the test as this one's identifiers will be the wrong case
-
-
-class BaseLowercasedSchemaTest(BaseTestSimpleCopy):
```

Comment:
So, I axed these only after trying to make them work for a while. The problem is that the way we override the scheme here is no longer an option because of how fixture objects are generated at runtime. 

I tried a lot of different hacks for manipulating the state of the project fixture but the connection object is already open. So it's very difficult to manipulate the schema name. I'm not sure how to override it at this point. I did try to override the unit texture but that doesn't really do it either because that's involved before test class.

And last, I wasn't even sure if these are very much worth the trouble anymore since quoting  rules are tested elsewhere